android/lib/network/annotation/get_annotation_service.dart

```diff
@@ -34,6 +34,33 @@ Future<List<ImageAnnotation>> getAnnotationsNetwork() async {
   }
 }
 
+Future<List<dynamic>> getTextAnnotationsNetwork() async {
+  Response response;
+
+  try {
+    response = await get(
+      Uri.parse(Uri.encodeFull(annotationsURL)),
+      headers: {'Content-Type': 'application/json; charset=UTF-8'},
+    );
+    if (response.statusCode != 200) {
+      return [];
+    }
+    List<dynamic> data = json.decode(response.body);
+    List<dynamic> textAnnotations = [];
+    for (var annotation in data) {
+      if (annotation['id'][0] == "c") {
```

Comment:
You should also update this if check after your update.

Reply:
I have made the requested changes in the commit https://github.com/bounswe/bounswe2022group7/commit/a98e85288c19e389f381ea0f93a6df5a7e7d492f.